Alignment/OfflineValidation/test/InputSource.dat

```diff
@@ -1,15 +1,22 @@
-jobname PVValidation
-isda False
+jobname PVValidation_pp2015B-HLTPhysics-EOY_alignment
+isda True   
+ismc False
+runcontrol True
+runnumber 251028
 applybows True
-applycorrs False
-datasetpath PVValidation_Example_cff
-maxevents 20000
-globaltag GR_E_V25::All
-alignobj frontier://PromptProd/CMS_COND_31X_ALIGNMENT
-taggeom TrackerAlignment_2009_v1_express
-bowsobj frontier://PromptProd/CMS_COND_310X_ALIGN
+applycorrs True
+datasetpath PVValidation_HLTPhysics2015B_TkAlMinBias_cff
+maxevents 10000
+globaltag auto:run2_data
+allFromGT False
+alignobj frontier://FrontierProd/CMS_CONDITIONS
+taggeom TrackerAlignment_v14_offline
+apeobj frontier://FrontierProd/CMS_CONDITIONS
+tagape TrackerAlignmentExtendedErr_2009_v2_express_IOVs 
+bowsobj frontier://FrontierProd/CMS_CONDITIONS
 tagbows TrackerSurafceDeformations_v1_express
-apeobj frontier://PromptProd/CMS_COND_31X_ALIGNMENT
-tagape TrackerAlignmentErr_2009_v2_express
 tracktype ALCARECOTkAlMinBias
-outfile PVValidation.root
\ No newline at end of file
+outfile PVValidation_pp2015B-HLTPhysics-EOY_alignment.root
+vertextype offlinePrimaryVertices
+lumilist /afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions15/13TeV/DCSOnly/json_DCSONLY.txt 
+ptcut 3.
```

Comment:
@mmusich no newline at the end of this file
btw., is this file actually still needed? or do you want to maintain it for standalone use?